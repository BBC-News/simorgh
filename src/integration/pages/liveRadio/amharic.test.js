/**
 * @service amharic
 * @pathname /amharic/bbc_amharic_radio/liveradio
 */

import runCrossPlatformTests from './crossPlatformTests';
import runAmpTests from './ampTests';
import runCanonicalTests from './canonicalTests';

describe('Live Radio Page', () => {
  describe(platform, () => {
    runCrossPlatformTests();
  });

  if (platform === 'amp') {
    describe(platform, () => {
      runAmpTests();
    });
  }

  if (platform === 'canonical') {
    describe(platform, () => {
      runCanonicalTests();
    });
  }
});
