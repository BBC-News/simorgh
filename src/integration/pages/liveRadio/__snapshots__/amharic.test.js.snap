// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Live Radio Page amp A11y I can see the skip to content link 1`] = `"ወደ ዋናው ይዘት ይለፉ"`;

exports[`Live Radio Page amp Core AMP AMP Consent config 1`] = `"{\\"consents\\":{\\"user-consent\\":{\\"promptIfUnknownForGeoGroup\\":\\"eea\\",\\"promptUI\\":\\"consent-prompt\\"}}}"`;

exports[`Live Radio Page amp Core AMP AMP Geo config 1`] = `"{\\"ISOCountryGroups\\":{\\"eea\\":[\\"at\\",\\"ax\\",\\"be\\",\\"bg\\",\\"bl\\",\\"cy\\",\\"cz\\",\\"de\\",\\"dk\\",\\"ea\\",\\"ee\\",\\"es\\",\\"fi\\",\\"fr\\",\\"gb\\",\\"gf\\",\\"gg\\",\\"gi\\",\\"gp\\",\\"gr\\",\\"hr\\",\\"hu\\",\\"ic\\",\\"ie\\",\\"im\\",\\"is\\",\\"it\\",\\"je\\",\\"li\\",\\"lt\\",\\"lu\\",\\"lv\\",\\"mf\\",\\"mq\\",\\"mt\\",\\"nc\\",\\"nl\\",\\"no\\",\\"pf\\",\\"pl\\",\\"pm\\",\\"pt\\",\\"re\\",\\"ro\\",\\"se\\",\\"si\\",\\"sj\\",\\"sk\\",\\"tf\\",\\"va\\",\\"wf\\",\\"yt\\"]}}"`;

exports[`Live Radio Page amp Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/amharic"`;

exports[`Live Radio Page amp Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. ቢቢሲ ከሌሎች ድረ-ገጾች ለሚመጡ መረጃዎች ሀላፊነት አይወስድም. ስለ ውጪ ሊንኮች ያለን አቀራረብ"`;

exports[`Live Radio Page amp Header I can see the branding 1`] = `"BBC News, አማርኛ"`;

exports[`Live Radio Page amp I can see the audio player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/media/bbc_amharic_radio/liveradio/am/amp"`;

exports[`Live Radio Page amp I can see the headline 1`] = `"ያድምጡ"`;

exports[`Live Radio Page amp I can see the summary 1`] = `"ዝግጅቶቻችንን’"`;

exports[`Live Radio Page amp Media player A11y I can read the media player title 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/media/bbc_amharic_radio/liveradio/am/amp"`;

exports[`Live Radio Page amp SEO Canonical link 1`] = `"http://localhost:7080/amharic/bbc_amharic_radio/liveradio"`;

exports[`Live Radio Page amp SEO Dir attribute 1`] = `"ltr"`;

exports[`Live Radio Page amp SEO FB admins 1`] = `"100004154058350"`;

exports[`Live Radio Page amp SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Live Radio Page amp SEO Heading level 1 1`] = `"ያድምጡ"`;

exports[`Live Radio Page amp SEO Lang attribute 1`] = `"am"`;

exports[`Live Radio Page amp SEO OG description 1`] = `"ዝግጅቶቻችንን’"`;

exports[`Live Radio Page amp SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/amharic.png"`;

exports[`Live Radio Page amp SEO OG image alt 1`] = `"BBC News አማርኛ"`;

exports[`Live Radio Page amp SEO OG locale 1`] = `"am-ET"`;

exports[`Live Radio Page amp SEO OG site name 1`] = `"BBC News አማርኛ"`;

exports[`Live Radio Page amp SEO OG title 1`] = `"ያድምጡ - BBC News አማርኛ"`;

exports[`Live Radio Page amp SEO OG type 1`] = `"website"`;

exports[`Live Radio Page amp SEO OG url 1`] = `"http://localhost:7080/amharic/bbc_amharic_radio/liveradio"`;

exports[`Live Radio Page amp SEO Page title 1`] = `"ያድምጡ - BBC News አማርኛ"`;

exports[`Live Radio Page amp SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Live Radio Page amp SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Live Radio Page amp SEO Twitter creator 1`] = `"@bbcnews"`;

exports[`Live Radio Page amp SEO Twitter description 1`] = `"ዝግጅቶቻችንን’"`;

exports[`Live Radio Page amp SEO Twitter image alt 1`] = `"BBC News አማርኛ"`;

exports[`Live Radio Page amp SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/amharic.png"`;

exports[`Live Radio Page amp SEO Twitter site 1`] = `"@bbcnews"`;

exports[`Live Radio Page amp SEO Twitter title 1`] = `"ያድምጡ - BBC News አማርኛ"`;
